src/store/new-request-module.ts

```diff
@@ -845,6 +845,7 @@ export class NewRequestModule extends VuexModule {
   tabNumber: number = 0
   userCancelledAnalysis: boolean = false
   waitingAddressSearch: WaitingAddressSearchI | null = null
+  isLoadingSubmission = false
```

Comment:
OK, this and the Mutation below will conflict with you. Please adjust accordingly.